src/core/utils/helpers/general.ts

```diff
@@ -405,6 +405,44 @@ export const filterLoansNotOverdue = (loans: LoanType[], warning: number) => {
   });
 };
 
+function getDateFromCpr(cprInput: string) {
+  const cpr = cprInput.replace(/[^\d]/g, "");
+  const dateSegments = cpr.substring(0, 6).match(/.{1,2}/g);
+
+  if (dateSegments) {
+    const [day, month, year] = dateSegments;
+    let prefix = "";
+    if (Number(year) < 21) {
+      prefix = "20";
+    } else {
+      prefix = "19";
+    }
+    const yearWithPrefix = Number(`${prefix}${year}`);
+
+    return new Date(
+      Date.UTC(
+        Number(yearWithPrefix),
```

Comment:
Should not be necessary to cast `yearWithPrefix` to an integer twice.

Reply:
fixed